src/ValiAddon.bas

```diff
@@ -297,30 +309,31 @@ Sub PushValis()
             Response = ValiAPI("rest/valis/" & ValiID, "PATCH", Data)
             'MsgBox (Response)
             Set JSONResponse = JsonConverter.ParseJson(Response)
+            MsgBox JSONResponse("name") & JSONResponse("name") & JSONResponse("project")
```

Comment:
Is this a debugging leftover?

Reply:
Might be, what is your take on it? @cauenapier

Reply:
That MsgBox was a debugging left over.